### ISSUE TITLE
Please change the project name

### ISSUE THREAD

**Issue by jellysquid3** [Someone-Else-Was-Taken/CaffeineMC-Reforged#25]
Hi.

I've asked that people porting our mods please do not re-use the original project names, as it causes player confusion and muddies the conversation around the original mod.  There's a longer version of the reasoning on my FAQ page [here](https://github.com/CaffeineMC/caffeine-meta/wiki/FAQ#where-are-the-forge-versions-of-your-mods).

This is not the first time I've tried to reach out to port authors to ask this, [as was the case with Halogen](https://github.com/spoorn/sodium-forge/issues/111#issuecomment-904145116). That issue also contains some additional reasoning for the request, and the respective maintainers seemed to be happy to change the name.

Ultimately, it's your project, and what name you want to use it up to you -- I'm just asking that it isn't referred to as "Sodium", "Sodium for Forge", or any other variant of that. 

Thanks in advance.

**Comment by viceversagames** [comment]
I agree. Name suggestion, **_Unofficial Forge Sodium Port Reforged For The Forge Launcher, and Definitely Not Fabric_.** This should clear up all confusion.

**Comment by jellysquid3** [comment]
@viceversagames I do not understand why you have continued antagonizing both me and other mod authors on this issue tracker. Rather than leaving snarky, toxic comments, could you either engage with the issue at hand and give me a reason why you believe my request is unreasonable, or just stop engaging with projects you seemingly have no holdings in?

**Comment by viceversagames** [comment]
You guys have made four issues so far complaining about this port. Rather than helping, bug fixing, providing advice, or supporting Forge yourselves, you've decided to antagonize someone who is taking it upon themselves to do so.

The mod is literally Sodium ported to Forge. "Sodium Reforged" is a perfectly fine name and it is marked as such and labelled as an unofficial port. All the other ports are similarly marked. It is not the port author's fault that there are idiots who cannot read.

If you're upset about the potential quality of the mod then perhaps get off your fucking high horse and contribute to the port that people clearly want. Making four useless issues to complain about the name does nothing.

**Comment by IMS212** [comment]
We would like to request the same thing for Iris, as stated in the readme of Iris.

**Comment by IMS212** [comment]
@viceversagames It’s been made very clear that the Forge developers have no intent on supporting jellysquid, or core mods in general. Alongside this, many mod developers hardcode for OptiFine, which breaks many assumptions for Iris and in some cases, Sodium.

**Comment by IMS212** [comment]
As well as this, I don’t believe much thought has been put into Iris for forge. I’ve already made a fully functional forge port privately… except for the part where 50% of mods don’t work due to them assuming Optifine is the only shaders mod.

**Comment by jellysquid3** [comment]
> Rather than helping, bug fixing, providing advice, or supporting Forge yourselves, you've decided to antagonize someone who is taking it upon themselves to do so.

I am asking the bare minimum for people to respect my work, which is to not use my branding, and to provide users with a way to get support if need be. If my mod is so highly sought after, people can take the minute of consideration to do this.

> If you're upset about the potential quality of the mod then perhaps get off your [expletive]...

These ports are never straight forward -- they need a lot of work -- and the result of that is they tend to often diverge heavily from upstream, so much so that it muddles the name significantly. That is all which is being said here. It has been true in the case of every port up to this point, and it will likely continue being true. That doesn't mean it's a bad port or whatever, just that it's different. Things which are different should be called different things. 

> and contribute to the port that people clearly want.

It is not my responsibility to port the mod to every platform. I am one person, doing the work that is normally done by entire teams of rendering engineers. I chose an open source license so that this conversation would even be possible. I really do not feel as if I'm asking for much in return.

> Making four useless issues to complain about the name does nothing.

Again, I am one person, who has made one issue. You are looking at that issue. My only other comment on this issue tracker was in direct response to a misrepresentation you made about me, which I view as fair game to respond to.

**Comment by DBotThePony** [comment]
> I agree. Name suggestion, **_Unofficial Forge Sodium Port Reforged For The Forge Launcher, and Definitely Not Fabric_.** This should clear up all confusion.

Magnesium-Forge

---------------------
Also being overwhelmingly hostile towards creator of *open-source* and *free* mod, which you can *legally* port, who spent their countless human-hours working hard, just because they want you to change one little thing does not put you into good standing.

**Comment by viceversagames** [comment]
Sounds like an PR needs to be made with Flywheel then. 

I think the fact that there are failed ports by third parties rather than official Forge versions is more "damaging to your reputation."

**Comment by asiekierka** [comment]
> I do not appreciate their vendetta against Forge. I get the feeling that they simply want Fabric to be exclusively better because of their technical and personal problems with Forge.

I literally made Fabric in part because of the technical and personal problems with Forge.

>Why would you not want to optimize Forge so it doesn't run like dogshit?

Because I don'

What JellySquid and co are requesting is absolutely reasonable. The Firefox web browser, for instance, forbids making changes without also changing the branding (name, logo, etc).

**Comment by Mino-dev** [comment]
> I appreciate their work otherwise I wouldn't be here. I do not appreciate their vendetta against Forge. I get the feeling that they simply want Fabric to be exclusively better because of their technical and personal problems with Forge. **Many people maintain Forge and Fabric versions without this drama.**

I don't think you're giving importance to the amount of work at all when you're trying to imply that they dropped Forge just because of the drama. These guys spent thousands of hours to work on improving the game on a tool chain they believed supports their goals, they are comfortable to work with. 

> They're going to go through all the effort of writing the core logic, but leave the Forge compatiblity up to other people under a different name? This is Sodium for Forge. If they want to pick the name for the forge port they should fucking port it themselves, or at least support it in any way. Really, I don't give a shit what the name is, this isn't my project, if it's named Magnesium or fucking CockAndBallTorture Reforged I don't care as long as it works. But that is up to the port owner, he is 100% allowed to continue calling it Sodium Reforged because that is what it is.

This is true, and they are requesting the owner of this port to change it with valid reasons why they felt it should be changed. Are they forcing the author to stop porting the projects to Forge? I don't think so. Why do you think that's the case anyway?

> I'm particularly hostile now because when I brought up the fact that I "don't think their internal politics should judge who gets to play with which mods" and I get labelled as a member of the "Forge community" and get called entitled. They are the **ones who dropped support for everyone** over their disagreements.

Why do you think everyone has to have a slice of someone else's cake? They don't owe you anything.

> I understand that Forge and the Forge team are terrible, and it would be nice if more mods were cross compatible, because Fabric is great. But the overwhelming majority of modpack players are stuck on Forge, and Fabric is painfully disappointing right now.

Again, the author has the right not to support platforms that they don't want to work with. They **shouldn't be forced** to support something that they don't want to support, especially if the reasons are valid. You think the reasons they stated above isn't? Why? 

> As it stands, these optimization mods are largely wasted on Fabric. Optifine without shaders mostly runs fast enough for vanilla. Fabric has no major content-based modpacks beyond vanilla+ tweaks and is missing many AAA mods that make the packs worth playing.

If you look into the perspective of a player who uses fabric, then no, that is not the case. As you stated, Forge community already has enough modpacks available. I don't get why that gives you the right to demand everything you want in Fabric to be ported to Forge.

> Why would you not want to optimize Forge so it doesn't run like dogshit? Why go through **all this effort just to blow it and not support half the playerbase**?

That's not the intention of this issue, no? Why do you think they are trying to stop this port as a whole? 

If you've been on their discord server you would understand, some users of these third party ports are going to CaffeineMC discord to get support, hence they want the distinction.

**Comment by viceversagames** [comment]
> some users of these third party ports are going to CaffeineMC discord to get support, hence they want the distinction

then tell them to stop being retarded, learn to read, and ban them.

**Comment by grondag** [comment]
> then tell them to stop being retarded, learn to read, and ban them.

Two things:

1) Using the "r" word reveals a lot about the quality of your character, and it's not good.
2) All of your responses amount to asking _other_ people to work harder to please you.

Why do you think any of us want to work harder on behalf of an insensitive asshat?

**Comment by Mino-dev** [comment]
> Forge team being bitches doesn't prevent you from coding with their toolchain.
Yes, because that's not the ultimate reason why they don't use it.

**Comment by viceversagames** [comment]
> Did you just call the Forge team more insults?

Would you rather I replace it with "[expletive]"? You afraid of getting cancelled or something? If they're being [expletive]s, call them [expletive]s.

> Is there anyone you respect in this universe?

idk Bob Ross was pretty cool

**Comment by asiekierka** [comment]
> You afraid of getting cancelled or something?

I already renamed myself to Hatsune Miku on Discord, you're too late.

I just don't think you should be insulting people for the sin of "not being educated on the intricate politics of Minecraft modding enough to know that 'Sodium Reforged' and 'Sodium' are two very different things with a not-entirely-supportive relationship between them" or "maintaining Minecraft Forge, a project I have many disagreements with, both technical and social, but ultimately respect the amount of work they've put into fostering a mod loader ecosystem".

**Comment by Nuclearfarts** [comment]
and nobody signed up for supporting people who are actually using a different mod with a similar name.

**Comment by asiekierka** [comment]
Your first mistake is that you think we treat mods as "products". Most of us don't. These are hobby projects. We do them out of passion. The amount of money almost any Minecraft modder gets from modding (outside of things like shader packs and Spigot plugins - we're talking Forge/Fabric here, which is a tiny subset of the overall Minecraft modding scene) is wildly disproportionate to the amount of labor put in and how much matching labor is worth to the commercial industry in just about any first-world region (and, with remote work being an option, many third-world regions as well).

As such, it pains us to see when our voices, as modders, aren't being listened to. It's a hobby - we can drop it at a moment's notice, because many of us do actually have industry jobs and would really not mind losing whatever tiny income we get from modding on the side. (The time spent could be used to do more industry work, anyway.)

Your second mistake is going all "my 15k discord" on us. I, too, have ran a Discord server. Not a 15k one, mind, a 10k one. After all, I was involved in the Fabric project for some time, and I, too, had to encounter a lot of confused users. The thing is, though, this stuff ISN'T EASY. It's NOT OBVIOUS. I have a neighbor - an eleven-year-old kid who really enjoys watching Minecraft YouTubers. They will often play with mods or shader packs. They will also often not give you any advice on where to find what they're using, leaving young teenagers stranded and forced to do the research themselves, without the necessary education or experience to know that 9minecraft isn't the correct place to get mods from.

We, who are ourselves programmers and developers, often forget that this stuff isn't immediately obvious, and that a lot of our users are genuinely inexperienced, and it's good to get some perspective at times. While I have been often frustrated by many members of the Fabric project's Discord, I don't consider them insult-worthy, as you've probably implied they are to you.

Honestly, given the amount of shit JellySquid has received which I have seen myself, I think she should just quit modding. She has a comfortable job with Hypixel Studios, where they probably treat her much better than the modding scene ever will. And it does pain me to say that. At least then you'll have your God-given right to push for keeping the mod's name as "Sodium Reforged" without opposition, and I think you might find that to be more important to you than the mod itself and the author behind it.

**Comment by viceversagames** [comment]
Yeah I mean I understand their POV. I just think them dropping Forge support is dumb, and if they don't want to support it themselves under the Sodium name (even a little), then they shouldn't be mad that others are trying to.

**Comment by viceversagames** [comment]
> Then stop posting on this thread as this should not concern you.

![image](https://user-images.githubusercontent.com/67132971/136882326-8c6a587a-70b1-4890-90e9-5032db3f36f8.png)

**Comment by IMS212** [comment]
😕

**Comment by IMS212** [comment]
Did the dev just suddenly go AWOL after this was posted? There’s been no activity at all.

**Comment by Return-To-Null** [comment]
> Did the dev just suddenly go AWOL after this was posted? There’s been no activity at all.

if they are awol it is your problem

**Comment by Return-To-Null** [comment]
> > > > This also isn't my mod lol I just have notifications on for the repo because I've been bug testing
> > > 
> > > 
> > > Then stop posting on this thread as this should not concern you.
> > 
> > 
> > this thread does not concern you either
> 
> This does not concern you either.



> > > > This also isn't my mod lol I just have notifications on for the repo because I've been bug testing
> > > 
> > > 
> > > Then stop posting on this thread as this should not concern you.
> > 
> > 
> > this thread does not concern you either
> 
> This does not concern you either.

you act like this concerns you at all
if you can be here I can too
